### PR TITLE
Add onerror example

### PR DESCRIPTION
**Pull request by beaufortfrancois**
This PR addresses comment posted by @reillyeon in https://github.com/w3c/web-nfc/pull/432#issuecomment-550478360

***
<a href="https://pr-preview.s3.amazonaws.com/beaufortfrancois/web-nfc/pull/439.html" title="Last updated on Nov 18, 2019, 1:51 PM UTC (c5109bc)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3c/web-nfc/439/6b0cdb8...beaufortfrancois:c5109bc.html" title="Last updated on Nov 18, 2019, 1:51 PM UTC (c5109bc)">Diff</a>

### CODE REVIEW

**Review by zolkis — APPROVED** [review]

File: index.html
Comment (kenchris) [inline index.html]: shows when sounds weird... shows how to handle? shows what happens?

**Comment by beaufortfrancois** [comment]
Let's wait for @reillyeon to review before merging.

File: index.html
Comment (reillyeon) [inline index.html]: The log message below says "try again" but this case is really the one were retrying makes sense (tell the user to hold their device against the tag for longer) whereas an error below indicates a more fatal issue.

Comment (beaufortfrancois) [reply index.html]: The onerror event is raised only for `IOException` and `FormatException`, not `TagLostException`. I believe* this means holding device longer won't change anything. Both are fatal errors.
Do you want to me remove "try again" to make it clearer?

Throws | Reason
-- | --
TagLostException | if the tag leaves the field
IOException | if there is an I/O failure, or the operation is canceled
FormatException | if the NDEF Message on the tag is malformed

Comment (reillyeon) [reply index.html]: Interesting. Do you think we should fire an event on `TagLostException`? That seems like the most likely and most actionable error that a site could encounter.

Given the current behavior I would word the error as, "Error! Cannot read data from the NFC tag. Try a different one?"

My goal is for this example code to demonstrate to developers what parts of their UI should be responding to different events, and how. This also gives us a good opportunity to test whether or not developers will be able to easily code against these potential errors.

Comment (beaufortfrancois) [reply index.html]: I don't think `TagLostException`provides that much value as users will try again if they see app is not reacting, while `IOException ` and `FormatException` are more "It's over" errors. They're nothing users can do about it as they use wrong formatted tags, and website will only show them something like "Sorry! This tag is not compatible with this application".

I'll update my example with your error message.

Comment (beaufortfrancois) [reply index.html]: See https://github.com/w3c/web-nfc/pull/439/commits/c5109bca16b5790ed3196dc2f11804ab445906e1

**Review by reillyeon — APPROVED** [review]

**Pull request merged**
